hnginternship5/ziki
# fixed the broken codes and added my updates

TITLE OF PR
Added updatePost function and changed the routing for single blog post

PURPOSE OF PR
To implement the functionality for updating posts and change the urls routing for single blog page

FEATURES IMPLEMENTED
update posts, simpler blog url e.g markessien.com/id instead of markessien.com/blog-details/id

HOW TO TEST
visit localhost:8000/test.php or alternatively create a post from the timeline, the former is recommended
after that locate the new file in the ./storage/contents folder e.g if the filename is 2233344.md, the visit localhost:8000/2233344 this should redirect you to the page .
TRELLO TICKET LINK
https://trello.com/c/lddFXOIH/4-single-blog-page-sphinx-be-cornflakes-fe-destiny-ihejirika

![working blog-details page](https://user-images.githubusercontent.com/41482784/57024736-3af56680-6c2d-11e9-971c-a3f23bed3acb.png)
